extensions/wikia/UserLogin/FacebookSignupController.class.php

```diff
@@ -162,6 +159,8 @@ public function modal() {
 			'returnToUrl' => $returnToUrl,
 			'queryString' => $returnToParams,
 			'loginToken' => UserLoginHelper::getSignupToken(),
+			'usernameKey' => UserLoginForm::SIGNUP_USERNAME_KEY,
+			'passwordKey' => UserLoginForm::SIGNUP_PASSWORD_KEY,
```

Comment:
Totally agreed. Garth and I actually discussed creating a ticket to clean up the back end of this extension, however since we're going to redo a lot of the user login and signup flow soon, we're going to forgo code optimizations like this for now.